website/project/views/contributor.py

```diff
@@ -378,6 +303,59 @@ def project_manage_contributors(auth, node, **kwargs):
     return {}
 
 
+@must_be_valid_project  # returns project
+@must_be_contributor
+@must_not_be_registration
+def project_remove_contributor(auth, **kwargs):
+    """Remove a contributor from a list of nodes.
+
+    :param Auth auth: Consolidated authorization
+    :raises: HTTPError(400) if contributors to be removed are not in list
+        or if no admin users would remain after changes were applied
+
+    """
+    contributor_id = request.json.get('contributorID')
```

Comment:
`request.json` is deprecated in flask (ref: http://flask.pocoo.org/docs/0.10/api/#flask.Request.json). Use `.get_json()` instead.